sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/display/DisplayData.java

```diff
@@ -306,9 +308,27 @@ private static Item create(ItemSpec<?> spec, Path path) {
           spec.getValue(), spec.getShortValue(), spec.getLabel(), spec.getLinkUrl());
     }
 
+    /** <b>For internal use only. No backwards compatibility guarantees.</b> */
+    @Internal
+    public static Item raw(
```

Comment:
Other ideas welcome for the best way to produce an `Item` directly - both for deserialization and for testing. I had tried extending `Item` in the translation class, and that works, but then I just make another `@AutoValue` class extending this one and it felt silly.